iml-api/src/graphql/mod.rs

```diff
@@ -485,6 +464,99 @@ impl QueryRoot {
 
         Ok(xs)
     }
+
+    #[graphql(arguments(
+        limit(description = "optional paging limit, defaults to all rows"),
+        offset(description = "Offset into items, defaults to 0"),
+        dir(description = "Sort direction, defaults to asc"),
+        message(description = "Pattern to search for in message"),
+        fqdn(description = "Pattern to search for in FQDN"),
+        tag(description = "Pattern to search for in tag"),
```

Comment:
Should note that this is using Postgres pattern matching (https://www.postgresql.org/docs/9.6/functions-matching.html)

Reply:
Done